toolsuite/src/main/java/com/faforever/neroxis/toolsuite/PbrTextureGenerator.java

```diff
@@ -26,21 +26,36 @@ public class PbrTextureGenerator implements Callable<Integer> {
     private CommandLine.Model.CommandSpec spec;
     @CommandLine.Mixin
     private OutputFolderMixin outputFolderMixin;
+    @CommandLine.Option(names = "--size", defaultValue = "1024", description = "Size of the input textures in pixels. Defaults to 1024.")
+    private Integer textureImageSize;
 
     @Override
     public Integer call() throws Exception {
+        if (!isPowerOfTwo(textureImageSize)) {
+            throw new IllegalArgumentException("Texture size must be a power of two!");
+        }
```

Comment:
Rather than checking on call you should make a setter and annotate the setter with the option and perform the check there.

Reply:
Why is that better?

Reply:
Because then you can throw the picocli parameter exception and it can print out the applicable help

Reply:
Also then we will know the value is never an invalid value